repos/spack_repo/builtin/packages/py_dask/package.py

```diff
@@ -108,6 +110,8 @@ class PyDask(PythonPackage):
     # Starting with version 2024.3.0, dataframe requires a separate package py-dask-expr
     depends_on("py-dask-expr", type=("build", "run"), when="@2024.3: +dataframe")
     depends_on("py-dask-expr@1.1.9", type=("build", "run"), when="@2024.7.1 +dataframe")
```

Comment:
These are no longer required

Reply:
Just waiting on these to have an upper bound

Reply:
It looks like `py-dask-expr` only has version `1.1.9`, but for `py-dask@2024.3.0` required `dask-expr@1.0`
https://github.com/dask/dask/blob/2024.3.0/pyproject.toml#L58
does it matter?

Otherwise I've fixed this thanks

Reply:
Commented below